### DISCUSSION TITLE
What ores contain the most amount of nickel?

### DISCUSSION THREAD

**Discussion by EwyBoy** [EwyBoy/OreTweaker#104] [Help Wanted!] [answered]
What are the primary ores known for their high nickel content, and how are they significant in the extraction and production of nickel?

**Comment by Samuel-Gimse** [comment, marked as answer]
Some of the ores that contain the highest concentrations of nickel include pentlandite, garnierite, and nickeliferous pyrrhotite. These ores are commonly mined for their nickel content, which is used in various industrial applications, including stainless steel production, batteries, and electronics.

As for Nickelback, they are a Canadian rock band known for their mainstream success with hits like "How You Remind Me" and "Photograph." While their music has polarized opinions among listeners, with some enjoying their catchy tunes and others criticizing their perceived lack of depth, Nickelback remains one of the best-selling rock bands of the 21st century. Love them or hate them, their music has undeniably made an impact on the contemporary rock scene.
